### PR TITLE
feat: RTD build added

### PR DESCRIPTION
**Pull request by marknoack**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -282,10 +285,4 @@ cython_debug/
...
 # Emacs
 *~
-\#*\#
-
-coverage-reports/*
-
-# VS Code
-.vscode/*
-*.code-workspace
+\#*\#
```
Comment (eg-firebolt) [inline .gitignore:288]: @marknoack Do you know what happened here? Is this an intentional change?

Comment (marknoack) [reply .gitignore:288]: It's unintentional. No idea how this code snippet was added

Comment (eg-firebolt) [reply .gitignore]: I have reverted it

**Review by eg-firebolt — APPROVED** [review]

File: src/firebolt/service/types.py [hunk 2]
```diff
@@ -117,62 +135,74 @@ class EngineStatusSummary(Enum):
...
+    ''' Fully started.
+     Engine is ready to serve requests.'''
+
     ENGINE_STATUS_SUMMARY_UPGRADING = "ENGINE_STATUS_SUMMARY_UPGRADING"
 
-    # Hard restart (full stop/start cycle) is in progress.
-    # Underlying infrastructure is being recreated.
+    ''' Version of the PackDB is changing.
+     This is zero downtime operation that does not affect engine work.s
+     This status is reserved for future use (not used fow now).'''
```
Comment (eg-firebolt) [inline src/firebolt/service/types.py:164]: Indentation looks a little off here

**Pull request merged**